common/src/mock.rs

```diff
@@ -327,7 +328,7 @@ macro_rules! mock_technical_config {
             type TechAccountId = TechAccountId;
             type Trigger = ();
             type Condition = ();
-            type SwapAction = ();
+            type SwapAction = pool_xyk::PolySwapAction<DEXId, AssetId, AccountId, TechAccountId>;
```

Comment:
I'm pretty sure that most of the pallets don't use SwapAction, and unit type can be used. Moreover, if we remove dependency on `pool_xyk`, it will improve build time. But this approach requires manual checking of every usage of `mock_technical_config`.

Reply:
I will check some pallets.

Reply:
Well, I will check

Reply:
Looks like all the pallets need it.